media-libs/freeglut/freeglut-3.0.0.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="alpha amd64 arm ~arm64 hppa ia64 ~mips ppc ppc64 ~s390 ~sh sparc x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
-IUSE="debug static-libs"
+IUSE="debug static-libs wayland gles"
```

Comment:
Keep this sorted. Also, if it explicitly requires ES 2.0 or newer, you might want to use `gles2` since that's what most of the packages do. `gles` alone is quite uncommon.